src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,267 @@
+import { MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { Events } from '../../lib/types/Enums';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Time } from '../../lib/util/constants';
+import { LLRCData, LongLivingReactionCollector } from '../../lib/util/LongLivingReactionCollector';
+import { getColor, resolveEmoji } from '../../lib/util/util';
+
+const enum HigherLowerReactions {
```

Comment:
shouldn't this be SCREAM_CASED?

Reply:
We use C# style here, not C++.

C# uses PascalCase, C++ uses SCREAMING_UPPER_CASE, which I dislike.